libmscore/score.cpp

```diff
@@ -2664,6 +2665,19 @@ void Score::cmdConcertPitchChanged(bool flag, bool /*useDoubleSharpsFlats*/)
 
             transposeKeys(staffIdx, staffIdx + 1, Fraction(0,1), lastSegment()->tick(), interval, true, !flag);
 
+            for (Segment* segment = firstSegment(SegmentType::Ambitus); segment; segment = segment->next1(SegmentType::Ambitus)) {
+                  //interval = staff->part()->instrument(segment->tick())->transpose();
+                  //if (!flag)
+                        //interval.flip();
+                  for (Element* e : segment->elist()) {
+                        if (e->isAmbitus()) {
```

Comment:
<code>e</code> can be a <code>nullptr</code>, so an extra test <code>e && ...</code> has to be added.

Reply:
isn't that implictly done in that `for( ... : ...)`

Reply:
Apparently not :wink: The first <code>for</code> iterates over staves, the second iterates over segments. I'm not sure this makes sense since a <code>Segment</code> contains similar elements of **all** staves.